index.js

```diff
@@ -36,6 +46,15 @@ Babel.prototype.constructor = Babel;
 Babel.prototype.extensions = ['js'];
 Babel.prototype.targetExtension = 'js';
 
+Babel.prototype.write = function(readTree, destDir) {
+  var self = this;
+  return Filter.prototype.write.call(this, readTree, destDir).then(function() {
+    if (self.exportModuleMetadata) {
+      fs.writeFileSync(destDir + path.sep + 'dep-graph.json', JSON.stringify(self.moduleMetadata, undefined, 2));
```

Comment:
is that because of ^ or... Seems like if this was stable, we could get away with not hashing.

also,`JSON.stringify` itself isn't stable, so if we continue the hashing approach, we should transition to a stable variant.